### PR TITLE
casper tests

### PR DESCRIPTION
**Pull request by cfalone**

### CODE REVIEW

**Comment by germanattanasio** [comment]
@cfalone The `README.md` in the uploads folder should not be removed. git doesn't track empty folders so we won't have uploads and that will break the demo

**Comment by cfalone** [comment]
Alright, added the readme back

**Comment by cfalone** [comment]
don't like my PR?

File: test/test.train-test.js [hunk 1]
```diff
@@ -61,6 +61,36 @@ casper.thenOpen('http://localhost:3000/train', function(result) {
...
+    casper.waitForSelector('div._examples[data-kind="moleskine"]', function() {
+
+    });
+
+    // 1 pro and one -non
+    casper.then(function() {
+      this.click('button._positive[data-name="portrait"]');
+    });
+    casper.then(function() {
+      this.click('button._negative[data-name="negative"]');
```
Comment (nfriedly) [inline test/test.train-test.js]: This one too.. I almost never use `.then()` in casper scripts.

But, OTHO, it works this way too, so not a big deal.

**Comment by nfriedly** [comment]
Looks good to me, merging

File: test/test.try.js [hunk 2]
```diff
@@ -6,63 +6,114 @@ casper.options.waitTimeout = 20000;
 casper.start();
 
 casper.thenBypassUnless(function() {
+  console.log(apiKey);
```
Comment (nfriedly) [inline test/test.try.js]: Oh, shoot, I missed this. This needs to go.

Comment (nfriedly) [inline test/test.try.js]: I killed the travis build before it got this far (which would have made it public) and removed the line in https://github.com/watson-developer-cloud/visual-recognition-nodejs/commit/bd586591414ab5eb3b11159b0ff43f904ed61ee2

Comment (cfalone) [inline test/test.try.js]: Yowser good catch!

**Pull request merged**